src/pages/Admin/Dashboard.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { IonContent } from '@ionic/react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+const InformationBlock = styled.div`
```

Comment:
pls consider removing the `Block` suffix, don't we risk to append this to a lot of components otherwise ?

Reply:
It's a copy/paste from `WelcomeCall`. I'll fix it

Reply:
Sorry, I just thought about that, not blocking anyway